src/net/RTP/MediaStream.cs

```diff
@@ -399,6 +488,52 @@ protected void SendRtpRaw(byte[] data, uint timestamp, int markerBit, int payloa
             }
         }
 
+        /// <summary>
+        /// To set a new value to a RTP Header extension.
+        /// 
+        /// According the extension the Object expected as value is different - check on each extension
+        /// </summary>
+        /// <param name="uri">The URI of the extension to use</param>
+        /// <param name="value">Object to set on the extension (check extension to know object type) </param>
+        public void SetRtpHeaderExtensionValue(String uri, Object value)

```

Comment:
Would a generic approach work better here? 

`SetRtpHeaderExtensionValue<T>(string uri, T value)`

Using Object is very broad. Can the value really be set to anything?

Reply:
Because each extension permits to manage different data, a specific object is expected according the extension:
- CVOExtension.CVO
- AudioLevelExtension.AudioLevel
- For AbsSendTime nothing to set

Another possibility will be to use a byte[] instead of an object. But it will be the same, you need to understand how to build it. Using an object (like CVOExtension.CVO) is easier I think.